Index: qaul.net/cmake/BuildLinux.cmake
===================================================================
--- qaul.net.orig/cmake/BuildLinux.cmake
+++ qaul.net/cmake/BuildLinux.cmake
@@ -1,6 +1,6 @@
 
 find_package (PkgConfig)
-pkg_check_modules (DBUS1 REQUIRED dbus-1)
+# pkg_check_modules (DBUS1 REQUIRED dbus-1)
 pkg_check_modules (UUID uuid)
 pkg_check_modules (OPENCORE_AMRNB opencore-amrnb)
 pkg_check_modules (OPENCORE_AMRWB opencore-amrwb)
